### PR TITLE
Update 06-AmsaleJonah.md

### PR DESCRIPTION
**Pull request by AmsaleJonah**
This is a bug list tracker for Release 11
Bug no 8/ resolution/and fix

*** Code changes made ****
Script update
Ne branching

To do:
Merge change to the branch
Review and update before applying

### CODE REVIEW

**Comment by AmsaleJonah** [comment]
Merge approved with the highlighted conflicts.

File: _slides/06-AmsaleJonah.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
----
-
-Caption here
-
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+Hi this is Amsale
```
Comment (tulikapgit) [inline _slides/06-AmsaleJonah.md:6]: amsale changed the file

**Review by tulikapgit — APPROVED** [review]

**Comment by AmsaleJonah** [comment]
Thank you, @tulikapgit

**Review by ClaireS-001 — COMMENTED** [review]
Looks good :)

**Pull request merged**